### ISSUE TITLE
Support drag/select multiple tab to a session

### ISSUE THREAD

**Issue by qqldd** [ReDEnergy/SessionSync#75]
I have the website list like:

a.com
  a.com/foo
  a.com/bar
  a.com/...

I add a session called a.com and would like to add all a.com/xxx to it. I tried shift+click, it  doesn't work.

**Comment by ReDEnergy** [comment]
Yes, I know :( ... quite annoying that I don't have this feature yet implemented. It's a planned feature for quite some time as it can be seen in #40 or #9. I really want to have it implemented and at some point it will be for sure but it is quite complex to do and I didn't really found the time till now (or I had other features/bugs more important to do first). I can't really promise  when it will be ready other than It will be implemented for sure at some point.

**Comment by th0rgall** [comment]
+1 for this. It's the main thing I'm missing in the extension, but of course I understand your time is limited. Thanks for the work!
